rln/src/pm_tree_adapter.rs

```diff
@@ -50,7 +50,7 @@ impl Hasher for PoseidonHash {
     }
 }
 
-fn get_tmp_path() -> PathBuf {
+pub fn get_tmp_path() -> PathBuf {
```

Comment:
This is a very bad & insecure way to create a temporary file. Better use tempfile (using with_prefix) here no?